src/Provider/TaxRateProvider.php

```diff
@@ -5,26 +5,20 @@
 namespace Sylius\RefundPlugin\Provider;
 
 use Doctrine\Common\Collections\Collection;
-use Sylius\Component\Core\Model\AdjustmentInterface;
 use Sylius\Component\Core\Model\OrderItemUnitInterface;
+use Sylius\RefundPlugin\Entity\AdjustmentInterface;
 
-final class LabelBasedTaxRateProvider implements TaxRateProviderInterface
+final class TaxRateProvider implements TaxRateProviderInterface
 {
     public function provide(OrderItemUnitInterface $orderItemUnit): ?string
     {
         /** @var Collection|AdjustmentInterface[] $taxAdjustments */
         $taxAdjustments = $orderItemUnit->getAdjustments(AdjustmentInterface::TAX_ADJUSTMENT);
 
-        if ($taxAdjustments->isEmpty()) {
+        if ($taxAdjustments->isEmpty() || !key_exists('taxRateAmount', $taxAdjustments->first()->getDetails())) {
```

Comment:
Should we refactor it in some way, to handle more than one tax adjustment? Or trigger an exception if there is more than one?

Reply:
I vote for triggering an exception, IMO there is no possibility to have 2 tax adjustments on one order item( @CoderMaggie what do you think about it?)

Reply:
As it is possible in the backend to have more than one tax adjustment, then we should handle all of them, not just one. 
Even if it is currently not possible to apply more tax adjustments in the core system, one might have that customized this way.